src/app/exercise/services/visitor.ts

```diff
@@ -0,0 +1,16 @@
+import * as ts from 'typescript';
+
+export function simpleVisitor(code, filter, callback) {
```

Comment:
done
test would be a pain to write (because would have to mock ts props) and refactoring this wouldn't be worth it, as we're most likely end up using tspoon instead.